app/code/Magento/WishlistGraphQl/Model/Resolver/UpdateProductsInWishlist.php

```diff
@@ -51,25 +55,49 @@ class UpdateProductsInWishlist implements ResolverInterface
      */
     private $wishlistFactory;
 
+    /**
+     * Catalog product
+     *
+     * @var Product
+     */
+    private $catalogProduct;
+
+    /**
+     * @var array
+     */
+    private $errors = [];
+
+    /**
+     * BuyRequestBuilder
+     * @var BuyRequestBuilder $buyRequestBuilder
+     */
+    private $buyRequestBuilder;
+
     /**
      * @param WishlistResourceModel $wishlistResource
      * @param WishlistFactory $wishlistFactory
      * @param WishlistConfig $wishlistConfig
      * @param UpdateProductsInWishlistModel $updateProductsInWishlist
      * @param WishlistDataMapper $wishlistDataMapper
+     * @param Product $catalogProduct
+     * @param BuyRequestBuilder $buyRequestBuilder
      */
     public function __construct(
         WishlistResourceModel $wishlistResource,
         WishlistFactory $wishlistFactory,
         WishlistConfig $wishlistConfig,
         UpdateProductsInWishlistModel $updateProductsInWishlist,
-        WishlistDataMapper $wishlistDataMapper
+        WishlistDataMapper $wishlistDataMapper,
+        Product $catalogProduct,
+        BuyRequestBuilder $buyRequestBuilder
     ) {
         $this->wishlistResource = $wishlistResource;
         $this->wishlistFactory = $wishlistFactory;
         $this->wishlistConfig = $wishlistConfig;
         $this->updateProductsInWishlist = $updateProductsInWishlist;
         $this->wishlistDataMapper = $wishlistDataMapper;
+        $this->catalogProduct = $catalogProduct;
```

Comment:
If we use hepler, it makes sense to rename this property to `productHelper` or similar. `catalogProduct` suits more for a product model.